web/template/index.html

```diff
@@ -40,8 +41,20 @@ <h2>Mainflux Deployment Summary</h2>
             //create map object and set default positions and zoom level
             var map = L.map('map').setView([20, 0], 2);
             L.tileLayer('https://{s}.tile.osm.org/{z}/{x}/{y}.png', {attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'}).addTo(map);
+
+            function applyFilter(event) {
```

Comment:
@SammyOina Is it possible to show the selected time range above the List of Countries ?

Reply:
the date is retained in the input now, so no need